src/pages/index.js

```diff
@@ -31,9 +33,24 @@ import {
   addProfileModal,
 } from "../utils/constants.js";
 
-//test
+/**************************************************************************
+ *                               API                                      *
+ **************************************************************************/
+//token: f4f8a497-25b7-4571-a952-d6bcf9aed847
 
-//validator .................................
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f4f8a497-25b7-4571-a952-d6bcf9aed847",
+    "Content-Type": "application/json",
+  },
+});
+
+//console.log(api);
+
+/**************************************************************************
+ *                               VALIDATION                               *
+ **************************************************************************/
 
 const editFormValidator = new FormValidator(
```

Comment:
**COULD BE IMPROVED**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```